solr/core/src/test/org/apache/solr/spelling/DirectSolrSpellCheckerTest.java

```diff
@@ -79,7 +79,7 @@ public void test() throws Exception {
       return null;
     });
   }
-  
+
```

Comment:
It's not clear to me what the "super" test above does. As far as I can see, the test runs a spellcheck for "super" but then uses "fob" as the index into suggestions, which will never find an entry.

Reply:
Yes, and it asserts that there are no results. It's the negative test case for the spell checking match.

Reply:
Good point. Indeed the test should be fixed line 77 to use spellOpts.tokens instead and expect empty suggestions.
Would you like to fix it?

Reply:
(comment race)

Reply:
I think it's clearer what's going on there now!